airflow/config_templates/config.yml

```diff
@@ -583,11 +583,13 @@ database:
       description: |
         Import path for connect args in SqlAlchemy. Defaults to an empty dict.
         This is useful when you want to configure db engine args that SqlAlchemy won't parse
-        in connection string.
-        See https://docs.sqlalchemy.org/en/14/core/engines.html#sqlalchemy.create_engine.params.connect_args
+        in connection string. This can be set by passing a dictionary containing the create engine parameters. 
+        For more details about passing  create engine parameters(keepalives variables, timeout etc) in postgres db connection 
+        see https://airflow.apache.org/docs/apache-airflow/stable/howto/set-up-database.html#setting-up-a-postgresql-database
+        Example: connect_args={"timeout":30} can be defined in airflow_local_settings.py and can be imported as shown below
       version_added: 2.3.0
       type: string
-      example: '{"timeout": 30}'
+      example: 'airflow.config_templates.airflow_local_settings.connect_args'
```

Comment:
Actually it's not right. It should be `airflow_local_settings.connect_args`. you are not supposed to use "config_templates" as package. It's what it is - set of confifuration templates. 

See one of the last notes in https://airflow.apache.org/docs/apache-airflow/stable/howto/set-up-database.html#setting-up-a-postgresql-database